hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(string)
+  middle = string.length / 2 # your code here
+
+  if string.length.odd?
+    string[middle]
+  else
+    string[middle - 1] + string[middle]
+  end
+end
+
+def run_cli
+  loop do
+    puts 'Enter string: '
+    string = gets.chomp
+
+    break if string == 'exit'
+
+    if string.empty?
+      puts 'String can not be blank!'
+    else
+      puts "Middle character: #{get_middle(string)}"
+    end
```

Comment:
Same with the ternary operator. Also, it can be a good style to extract pyre strings into constants. Mixed strings can be moved to the methods.